### PR TITLE
Source listing in bootstrap.s is incorrect.

### PR DESCRIPTION
**Pull request by jefftranter**
The OSI C1P alternative boot file format works, but the code in the source listing does not match the ASCII-coded hex translation (which is actually used). This is confusing to anyone trying to maintain the code.

Also, the source code does not assemble when ASM is defined.

With these changes the source file should correctly match what is used at run time.

### CODE REVIEW

File: libsrc/osic1p/bootstrap.s [hunk 1]
```diff
@@ -45,20 +45,22 @@ GETCHAR         :=      $FFBF           ; gets one character from ACIA
...
-        ldx     #(<load_size) + 1
-        stx     count
-        ldx     #(>load_size) + 1
-        stx     count+1                 ; save size with each byte incremented separately
+        lda     #<load_size
+        eor     #$FF
+        sta     count
+        lda     #>load_size
+        eor     #$FF
+        sta     count+1
```
Comment (mrdudz) [inline libsrc/osic1p/bootstrap.s:59]: Why not eor with $ffff at compile time?

**Comment by jefftranter** [comment]
I revised the pull request.

**Comment by mrdudz** [comment]
Just now i actually realise what this PR is trying to do... would'nt it make more sense to fix the ASCII listing at the bottom to contain the original code (which is more optimal!) rather than restoring the old (less optimal) code by translating back the ascii stuff?

**Comment by jefftranter** [comment]
I have to confirm, but my recollection was that the source version (not the ASCII listing) didn't work properly.

**Comment by mrdudz** [comment]
> I have to confirm, but my recollection was that the source version (not the ASCII listing) didn't work properly.

Well, but you could surely test - and if it doesnt work, fix - it? It's not very complex code afterall :)

**Comment by mrdudz** [comment]
looks good now, thanks!

**Pull request merged**